<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>691</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Fournitures </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sylfaen</family>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="title">
       <string>Ajouter un article</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="Num">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sylfaen</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QLabel" name="nom">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sylfaen</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="categorie">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sylfaen</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ref_fourniture">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_fourniture">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>150</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sylfaen</family>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Stock</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stock_article">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="categorie_article">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher Fournitures</string>
     </attribute>
     <widget class="QTableView" name="Tab_articles">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer Fournitures</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>681</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="title">
       <string>Supprimer article</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Reference</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ref_delete">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Supprimer">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion_Fournitures">
    <property name="title">
     <string>Gestion Fournitures</string>
    </property>
   </widget>
   <addaction name="menuGestion_Fournitures"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
